resources/assets/js/components/pages/admin/users/StaffManager.vue

```diff
@@ -55,11 +55,12 @@
                     </template>
                 </el-table-column>
                 <el-table-column align="right">
-                    <template slot="header">
+                    <template slot="header" slot-scope="scope">
```

Comment:
What does slot-scope do here?

Reply:
That carries the scope of the parent component to the child component. For the Vue component library Atlas is using it is a requirement for the table component. This is the fix for the not being able to type issue, no results was fixed by changing the default code from the examples.